<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensor Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="PlotWidget" name="fluidPlot"/>
    </item>
    <item>
     <widget class="QLabel" name="fluidValue">
      <property name="text">
       <string>fluid sensor: uninitialized</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PlotWidget" name="sensor0Plot"/>
    </item>
    <item>
     <widget class="QLabel" name="sensor0Value">
      <property name="text">
       <string>surface sensor 0: uninitialized</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PlotWidget" name="sensor1Plot"/>
    </item>
    <item>
     <widget class="QLabel" name="sensor1Value">
      <property name="text">
       <string>surface sensor 1: uninitialized</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PlotWidget" name="sensor2Plot"/>
    </item>
    <item>
     <widget class="QLabel" name="sensor2Value">
      <property name="text">
       <string>surface sensor 2: uninitialized</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PlotWidget" name="sensor3Plot"/>
    </item>
    <item>
     <widget class="QLabel" name="sensor3Value">
      <property name="text">
       <string>surface sensor 3: uninitialized</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="statusTip">
    <string>Quit the application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
